python/pip.bzl

```diff
@@ -235,6 +235,9 @@ def dist_info_requirement(name):
 def entry_point(pkg, script = None):
     fail("Not implemented yet")
 
+def entry_point_binary(name, pkg, script = None, **kwargs):
+    fail("Not implemented yet")
```

Comment:
Instead of simply "not implemented yet", lets change the message to better indicate that codegen generates the implementation for this, so if you hit this, it means you're loading the wrong thing.